tests/Zipkin/TraceRequestsTest.php

```diff
@@ -44,7 +44,7 @@ public function trace_http_requests()
         $reporter->shouldHaveReceived('report')->with(Mockery::on(function ($spans) {
             $span = $this->shiftSpan($spans);
 
-            $this->assertEquals('POST shipments/{id}', Arr::get($span, 'name'));
+            $this->assertEquals('POST /shipments/{id}', Arr::get($span, 'name'));
```

Comment:
I got an error with this change. I wonder if there some differences in the locally installed dependencies. Try running `composer install` and verify if you still need to add a forward slash here. I seem to remember that I removed it after Laravel version upgrade.

![image](https://user-images.githubusercontent.com/10194667/91646125-8fdd0300-ea54-11ea-9e00-4d957b6b5f21.png)

Reply:
Hello Alex, I did and this test still pass. Could be related to PHP version, I am running `PHP 7.1.33`. Which version are you running?
I presume we should create a `docker-compose` with a consolidated configs to avoid such situations.

Reply:
PHP 7.4.8. Yeah, could be related.

Reply:
I added docker-compose.yml file with `vinelab/nginx-php:composer` image that is able to execute tests without issues assuming you install dependencies using that version of PHP.

Reply:
Thank you, I was looking into this yesterday, but we realized that this failed tests highlights a backward compatibility issue.
According to `composer.json` this library supports `"php": "^7.1.3"` but its not actually the case since this test would fail for later version of php.

Reply:
I think it's best to also make it compatible with higher versions of PHP. Travis can help ensure they're supported moving forward but for the time-being in dev it would have to be done manually unless we add multiple `docker-compose` files?! Any other suggestions in that regard? 

@adiachenko on a side-note, there's [`vinelab/nginx-php:7.4`](https://hub.docker.com/layers/vinelab/nginx-php/7.4/images/sha256-dabc427480f68952fd1410c75ad466fb8581004964133c1a06bea9cdae3d8ac8?context=explore) with composer now 😄

Reply:
The issue seems to be limited to a single test case for now so we can simply fix that to support any PHP version from 7.1 onwards but I wonder if there is a better way to address this. 

It's just that we support too many versions at this point:
- PHP >= 7.1 (should be like >= 7.3 according to [official support page](https://www.php.net/supported-versions.php))
- Laravel >= 5.5 (should be like >= 6.0 according to [official support page](https://laravel.com/docs/master/releases#support-policy))

The best-case scenario would be to simply bump deps in the next major version of the package and then issues like these may just very well go away on their own. Until then I'll look into editing tests so they're more tolerant to changes in PHP version.